@trofi fails on aarch64-linux 

```
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/bmifv173i6vfql2ynv6qqxifykw4a7xz-re2-2022-02-01
shrinking /nix/store/bmifv173i6vfql2ynv6qqxifykw4a7xz-re2-2022-02-01/lib/libre2.so.9.0.0
strip is /nix/store/8wwqp5phmra2jwrm0j6mvr1kdi7s7g8v-gcc-wrapper-9.3.0/bin/strip
stripping (with command strip and flags -S) in /nix/store/bmifv173i6vfql2ynv6qqxifykw4a7xz-re2-2022-02-01/lib 
patching script interpreter paths in /nix/store/bmifv173i6vfql2ynv6qqxifykw4a7xz-re2-2022-02-01
checking for references to /build/ in /nix/store/bmifv173i6vfql2ynv6qqxifykw4a7xz-re2-2022-02-01...
running install tests
installcheck flags: -j64 -l64 SHELL=/nix/store/5a50a0gjsznslppq3wk4bvyg9bmfj4l5-bash-5.1-p12/bin/bash testinstall
cp: cannot create regular file 'obj/testinstall.cc': File exists
make: *** [Makefile:334: static-testinstall] Error 1
make: *** Waiting for unfinished jobs....
(cd obj && g++ testinstall.cc -o shared-testinstall -pthread -I/nix/store/bmifv173i6vfql2ynv6qqxifykw4a7xz-re2-2022-02-01/include -O3 -g -pthread -L/nix/store/bmifv173i6vfql2ynv6qqxifykw4a7xz-re2-2022-02-01/lib -lre2  )
LD_LIBRARY_PATH="/nix/store/bmifv173i6vfql2ynv6qqxifykw4a7xz-re2-2022-02-01/lib:" obj/shared-testinstall
PASS
error: builder for '/nix/store/5r0c3dax4mx7ayw9szffdkbpkaa4nygh-re2-2022-02-01.drv' failed with exit code 2;
```

--- a/pkgs/development/libraries/re2/default.nix
+++ b/pkgs/development/libraries/re2/default.nix
@@ -1,24 +1,15 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, nix-update-script }:
+{ lib, stdenv, fetchFromGitHub, nix-update-script }:
 
 stdenv.mkDerivation rec {
   pname = "re2";
-  version = "2021-11-01";
+  version = "2022-02-01";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "re2";
     rev = version;
-    sha256 = "sha256-q8u7xNp6n17F6twPoVkix8iCKPWUN+qg6rhSR4Dv+bI=";
+    sha256 = "sha256-3RspCfJD2jV7GYuzeBUcxkZsdHyL14kaz8lSoIrH7b8=";
   };
-
-  patches = [
-    # Pull upstreal fix for parallel testing.
-    (fetchpatch {
-      name = "parallel-tests.patch";
-      url = "https://github.com/google/re2/commit/9262284a7edc1b83e7172f4ec2d7967d695e7420.patch";
-      sha256 = "1knhfx9cs4841r09jw4ha6mdx9qwpvlcxvd04i8vr84kd0lilqms";
-    })
-  ];
 
   preConfigure = ''
     substituteInPlace Makefile --replace "/usr/local" "$out"
@@ -29,6 +20,11 @@ stdenv.mkDerivation rec {
   buildFlags = lib.optionals stdenv.hostPlatform.isStatic [ "static" ];
 
   enableParallelBuilding = true;
+  # Broken when shared/static are tested in parallel:
+  #   cp: cannot create regular file 'obj/testinstall.cc': File exists
+  #   make: *** [Makefile:334: static-testinstall] Error 1
+  # Will be fixed by https://code-review.googlesource.com/c/re2/+/59830
+  enableParallelChecking = false;
 
   preCheck = "patchShebangs runtests";
   doCheck = true;

